### ISSUE TITLE
Potion warning keeps spamming when using infinite duration

### ISSUE THREAD

**Issue by MrPondi** [maruohon/tweakeroo#445]
Just keeps informing that the effect will run out in 0 second over and over when the duration is set to infinite(-1).

![screenshot](https://github.com/maruohon/tweakeroo/assets/109785648/4e91cc9f-18f8-4ad1-9c8a-27a5d080d809)

**Comment by catthou** [comment]
Thank goodness someone else reported this. Pity there's bug reports open going back such a long distance though. Wish I knew java even a little so I could lend a hand.

**Comment by maruohon** [comment]
@catthou Yeah most of the open issues in the repos were meant to be worked on during the big mod rewrites and code cleanup. I worked on that stuff for ~3 years for as much as I had free time, mostly working on malilib changes and new features and got that into a pretty good state, but the other mods still need their own cleanup and refactoring. And then my available free time, and especially energy and motivation to work on the mods after my new IRL job plummeted, and I've been mostly in a total modding burnout lately.

So basically nothing major has been done for the 1.13+ mod versions since mid 2019, they have been mostly in minimal life support mode, with only minor additions here and there and porting the old junk code to new MC versions. Any big changes or big new features and new systems only go to the main development branch which is still in 1.12.2. And even those (except for malilib) haven't really had new features since mid 2020 when I started on the malilib refactors. So yeah, all in all the 1.13+ mod version have been pretty dead feature wise for like 5 years now. And sadly lots of these minor bug fixes that wouldn't take much time have also gotten neglected...

It remains to be seen if I regain motivation to continue properly working on the mods at some point. Lately I've been more interested in playing old beta and older versions of the game myself (not that I've actually played anything yet...), and thus I'd rather like to backport the new code version to those versions rather than working on new MC versions which don't really interest me... Plus I've started spending most of my free time to other stuff like getting back to some electronics projects. So yeah... :shrug:
